tests/test_attribute_definition.py

```diff
@@ -462,7 +462,7 @@ def print_parameter(self, printer: Printer) -> None:
         printer.print_string("]")
 
     @staticmethod
-    def generic_constraint_coercion(args: tuple[Any]) -> AttrConstraint:
+    def generic_constraint_coercion(args: tuple[Any, ...]) -> AttrConstraint:
```

Comment:
@math-fehr I'm not sure I did this right. The issue was that an n-tuple was passed to a one-tuple function. Should I have checked for length at the callsite instead?

Reply:
Yes, I think instead we should check at callsite, and this function should not assert (since it's in the type)